catwalk/models/utils_qa.py

```diff
@@ -0,0 +1,185 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Team All rights reserved.
```

Comment:
We can't have the huggingface header copy & pasted in here.

Reply:
If this is part of transformers, can we just call their version, and skip having our own?

Reply:
This is from the examples/pytorch/question-answering directory inside the transformers repo and cannot be called directly the library. What would be a good way to keep this but properly cite the source? Alternatively if we do choose to use a pipeline then we would not need this function.

Reply:
Got it. I use pipelines then.